lib/profile_screen/view/profile_page.dart

```diff
@@ -18,78 +18,65 @@ class _ProfilePageState extends State<ProfilePage> {
         child: LayoutBuilder(
           builder: (context, BoxConstraints box) {
             return Padding(
-              padding: const EdgeInsets.only(top: 25, left: 25, right: 25),
+              padding: EdgeInsets.only(
+                top: box.maxHeight * 0.005,
+                left: box.maxWidth * 0.05,
+                right: box.maxWidth * 0.05,
+              ),
               child: Column(
                 crossAxisAlignment: CrossAxisAlignment.start,
                 children: [
                   Text(
                     'Profile',
-                    style: Theme.of(context).textTheme.displayMedium?.copyWith(
-                          fontFamily: 'open sans',
-                          fontSize: 32,
-                          fontWeight: FontWeight.w600,
+                    style: Theme.of(context).textTheme.headlineMedium?.copyWith(
                           color: const Color(0xff181818),
```

Comment:
Colors should not be defined in the widgets. Make all colors global in the colors class.